halibot/message.py

```diff
@@ -26,12 +26,26 @@ def makeOrigin(self):
 
 class Message():
 
+	class Author(str):
+		def __init__(self, a, id=""):
+			self.id = id
+			self.name = str(a) if a else ""
+		def __str__(self):
+			return self.name
+		def __repr__(self):
+			return self.name
+		def __setattr__(self, key, value):
+			if key in ("id", "name"):
+				object.__setattr__(self, key, value)
```

Comment:
Should probably throw an exception in the else case, as it is an error on their part.